src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyGridTests.cs

```diff
@@ -4064,4 +4078,70 @@ public new bool VScroll
 
         public new void WndProc(ref Message m) => base.WndProc(ref m);
     }
+
+    private class ErroredCombo
+    {
+        [TypeConverter(typeof(ExpandableObjectConverter))]
+        public virtual ComboPath Path { get; set; } = new ComboPath() { };
+
+        public class ComboPath
```

Comment:
```suggestion
        public class MyExpandableClass
```
or a better name  :)